waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/ThriftMetastoreClientManager.java

```diff
@@ -167,6 +167,8 @@ void open() {
                     + CONN_COUNT.incrementAndGet());
 
             isConnected = true;
+            LOG.info("calling #set_ugi for user '{}',  on URI {}", ugiArgs.getUser(), store);
+            client.set_ugi(ugiArgs.getUser(), ugiArgs.getGroups());
```

Comment:
This bit is a new addition right? I would say adding here the set helps for the use case we discussed last week